openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -1377,11 +1399,24 @@ def create_openmm_system(self, topology, **kwargs):
             return system
 
     @requires_package("openff.interchange")
-    def _to_interchange(self, topology, box=None):
+    def to_interchange(self, topology: "Topology", box=None):
```

Comment:
Since this doesn't convert the forcefield inTO an `Interchange` object, but rather is a method that adds a lot of information content and CREATEs an `Interchange` object, what would you think about calling this `create_interchange`?

Reply:
Sure, seems like a pure improvement to me